import/api/wpforms-crawler/merge.sh

```diff
@@ -1,8 +1,14 @@
+#!/usr/bin/env bash
 ## quick and dirty! TODO: replace grep -v
+set -eu -o pipefail
 
-source ./config.sh
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+# shellcheck source=config.sh
+source "$SCRIPT_DIR/config.sh"
 
-(cd $DATA_DIR; head -n1 $(ls | head -n1)) > $OUT
-cat $DATA_DIR/* | grep -v 'Name,Land,Straße,Hausnummer' >> $OUT
+pushd "$DATA_DIR"
+find . -type f | head -n1 | xargs head -n1 >"$OUT"
```

Comment:
bitte nicht schon wieder :/ das macht es wieder genauso kaputt wie in 13caed2c3000082f18bb0bc8e39ba318a54d0cd1, dort habe ich es detailiert kommentiert

Reply:
Deswegen das `-type -f`.